mod/wikirate/set/right/related_articles.rb

```diff
@@ -13,25 +14,40 @@
           uncited << analysis_name
         end
       end
-
-      if cited.any?
+      if params[:general_overview] && params[:company]
+        body +=
+          content_tag :div,
+                      class: 'related-articles cited-articles related-overviews cited-overviews' do
+            card_link "#{params[:company]}+notes_page",
+                      text: 'Cite in General Overview',
+                      path_opts: {
+                        citable: claim.cardname.url_key,
+                        edit_general_overview: true
+                      },
+                      class: 'cite-button'
+          end
+      end
+      if uncited.any?
         body += %{
-          <div class="related-articles cited-articles related-overviews cited-overviews">
-            <h3>Overviews that cite this Claim</h3>
-            <ul>#{ cited.map { |a| "<li>#{ analysis_links a, :cited=>true }" }.join "\n" }</ul>
+          <div class="related-articles uncited-articles related-overviews cited-overviews">
+            <h4>Overviews that <em>could</em> cite this Claim</h4>
+            #{list_tag uncited.map { |a| analysis_links(a).html_safe }}
+              <h4>#{testparam}</h4>
           </div>
         }
       end
-      if uncited.any?
+      if cited.any?
         body += %{
-          <div class="related-articles uncited-articles related-overviews cited-overviews">
-            <h3>Overviews that <em>could</em> cite this Claim</h3>
-            <ul>#{ uncited.map { |a| "<li>#{ analysis_links a }" }.join "\n" }</ul>
+          <div class="related-articles cited-articles related-overviews cited-overviews">
+            <h4>Overviews that cite this Claim</h4>
+            <ul>#{ cited.map { |a| "<li>#{ analysis_links a, :cited=>true }" }.join "\n" }</ul>
```

Comment:
Line is too long. [95/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside string interpolation detected.